frontend/src/components/smart/QuestCharacter.vue

```diff
@@ -0,0 +1,230 @@
+<template>
+  <div class="quest-character-display">
+    <img class="quest-character-portrait m-4"
+         :class="'character-animation-applied-' + getCharacterCosmetic(character.id)"
+         :src="getCharacterArt(character)" alt=""/>
```

Comment:
Is alt required here?

Reply:
![image](https://user-images.githubusercontent.com/34208222/155973139-db7ced56-4554-47a8-8628-f4e9dd37362f.png)
Hinted by the ide, my bad that it's empty, I'll add some meaningful content there

Reply:
We have tons of <img instances without this attribute.